src/api/app/policies/bs_request_policy.rb

```diff
@@ -13,4 +13,11 @@ def handle_request?
     is_author = record.creator == user.login
     record.state.in?([:new, :review, :declined]) && (is_target_maintainer || is_author)
   end
+
+  def add_reviews?
+    is_target_maintainer = record.is_target_maintainer?(user)
+    is_author = record.creator == user.login
+    open_reviews = record.reviews.where(state: 'new').select { |review| review.matches_user?(user) }
```

Comment:
IIUC it is used only as a `.present?` boolean condition check. Maybe it could be more straight forward to save the boolean only: `has_open_reviews = record.reviews.where............present?` only

Reply:
done